lib/nexpose/scan.rb

```diff
@@ -763,6 +788,42 @@ def self._parse_status(code)
         :stopped
       when 'A'
         :aborted
+      when 'P'
+        :paused
+      when 'I'
+        :integrating
+      else
+        :unknown
+      end
+    end
+  end
+
+  class ActiveScan < CompletedScan
+    def self.parse_dyntable(json)
+      new do
+        @id = json['Scan ID']
+        @site_id = json['Site ID']
+        @status = CompletedScan._parse_status(json['Status Code'])
+        @start_time = Time.at(json['Started'].to_i / 1000)
+        @end_time = Time.at(json['Progress'].to_i / 1000)
```

Comment:
Do not use `Time.at` without zone. Use one of `Time.zone.at`, `Time.at.current`, `Time.at.in_time_zone`, `Time.at.utc`, `Time.at.getlocal`, `Time.at.iso8601`, `Time.at.jisx0301`, `Time.at.rfc3339`, `Time.at.to_i`, `Time.at.to_f` instead.